e2e/tests/policy-forms.spec.ts

```
@@ -487,18 +459,551 @@ test.describe('other policy create pages render', () => {
     await expect(page.getByRole('button', { name: 'Cancel' })).toBeVisible();
   });
 
-  test('TokenRateLimitPolicy create page renders YAML editor', { tag: '@smoke' }, async ({ page }) => {
+  test('TokenRateLimitPolicy create form renders with required fields', { tag: '@smoke' }, async ({ page }) => {
```

ESLint

[error] Replace `'TokenRateLimitPolicy·create·form·renders·with·required·fields',·{·tag:·'`@smoke`'·},` with `⏎····'TokenRateLimitPolicy·create·form·renders·with·required·fields',⏎····{·tag:·'`@smoke`'·},⏎···`

(prettier/prettier)